test/controllers/sections/guarantor/GuarantorNameControllerSpec.scala

```diff
@@ -152,7 +152,7 @@ class GuarantorNameControllerSpec extends SpecBase with MockUserAnswersService {
 
         status(result) mustEqual SEE_OTHER
         redirectLocation(result).value mustEqual
-          controllers.sections.guarantor.routes.GuarantorArrangerController.onPageLoad(testErn, testLrn, NormalMode).url
+          controllers.sections.guarantor.routes.GuarantorIndexController.onPageLoad(testErn, testLrn).url
```

Comment:
The test name says redirect to 'arranger controller' I think this should be 'index controller'